packages/lesswrong/components/comments/CommentsTimeline.tsx

```diff
@@ -52,8 +52,18 @@ const CommentsTimelineFn = ({
   // Scroll to the bottom when the page loads
   const currentHeight = bodyRef.current?.clientHeight;
   useEffect(() => {
-    if (!userHasScrolled && bodyRef.current)
+    if (!userHasScrolled && bodyRef.current) {
       bodyRef.current?.scrollTo(0, bodyRef.current.scrollHeight);
+
+      // For mobile: scroll the entire page to the bottom to stop the address bar from
+      // forcing the bottom off the end of the page. On desktop this just does nothing
+      // because the page fills the screen exactly
+      window.scrollTo({
+        top: 500,
+        // 'smooth' is to try and encourage it to scroll the address bar off the scree rather than overlaying it
+        behavior: 'smooth'
+      });
```

Comment:
This is still not an ideal solution, but I think it's the best we can do. I looked in to using the the [fullscreen API](https://web.dev/native-hardware-fullscreen/) but it wasn't really appropriate for this